src/content/configuration/dev-server.md

```diff
@@ -865,17 +865,20 @@ Specify a page to navigate to when opening the browser.
 module.exports = {
   //...
   devServer: {
-    openPage: '/different/page',
+    open: true,
+    openPage: 'different/page',
   },
 };
 ```
 
 Usage via the CLI
 
 ```bash
-npx webpack serve --open-page /different/page
+npx webpack serve --open --open-page different/page
 ```
 
+W> You do not need to add `/` in starting webpack-dev-server adds it automatically for you else the browser will open `http://localhost:8080//different/page`
```

Comment:
```suggestion
W> Do not prepend `/` for the page as webpack-dev-server will do it automatically, otherwise the browser will open urls like `http://localhost:8080//different/page`.
```

Reply:
Also fixed in webpack-dev-server@beat ^^

Reply:
Good to know, we should remove this warning once webpack-dev-server 4 is released.